packages/html/ds/src/link/link.schema.ts

```diff
@@ -1,10 +1,11 @@
 import * as zod from 'zod';
 import { buttonSchema } from '../button/button-schema';
+import { getEnumValues } from '../helpers';
 
-export enum LinkSize {
-  SMALL = 'sm',
-  MEDIUM = 'md',
-}
+export const LinkSize = {
+  SMALL: 'sm',
```

Comment:
Check that all the "small" are named `sm` and not `SMALL` or `Small`.
This comment applies to all the enums.

Reply:
As this PR is becoming huge I think we can have a separated PR to deal with the "enum" values.

All the "enum" keys are in uppercase so we have a pattern to follow, we can address the values changes in a separeted PR.

what do you think?